src/schema/users.schema.ts

```diff
@@ -4,6 +4,7 @@ import { Types } from 'mongoose';
 
 export enum UserRole {
   Owner = 100,
+  Chef = 75,
```

Comment:
Why have Chef's in UserRole, as we discussed chef == employee

Reply:
Okay but in README.md Chef is 75